src/components/Table.jsx

```diff
@@ -143,31 +115,119 @@ export default function Table() {
       }
     };
-    setLoader(true);
-    fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}`, {
-      method: "POST",
-      body: JSON.stringify(body),
-      headers: {
-        "Content-type": "application/json; charset=UTF-8"
+      const response = await fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}`,
+       {
+        method: 'POST',
+        body: JSON.stringify(body),
+        headers: {
+          "Content-type": "application/json; charset=UTF-8"
+        }
       }
-    })
-      .then((response) => response.json())
-      .then((result) => {
-        setCustomers(result?.data);
-        setLoader(false);
-      });
+  );
+  const data = await response.json();
+  if(data?.success){
+    setCustomers(data?.data);
+    setLoader(false)
+  }
+  if (!data?.success){
+    console.log("data>>",data?.error)
+	}
+}
+
+async function getBrands(){
+const response = await fetch(`${workFlowsUrl}/search?limit=${limit}&page=${currentPage}&unique=brand`,
+       {
+        method: 'POST',
+        headers: {
+          "Content-type": "application/json; charset=UTF-8"
+        }
+      }
+  );
+  const data = await response.json();
+  if(data?.success){
+    setBrands(
+      data?.data?.uniqueValues
+        .filter(Boolean)
+        .map((item) => ({ brand: item }))
+    )
+  }
+  if (!data?.success){
+    console.log("data>>",data?.error)
+	}
+}
+
+async function getStatus(){
+  const status = [];
+  switch (currentTab) {
+    case "Unassigned":
+      status.push("WAITING_FOR_WRITER");
+      break;
+    case "Completed":
+      status.push("WRITING_COMPLETE", "EDITING_COMPLETE");
+      break;
+    case "Assigned":
+      status.push("ASSIGNED_TO_WRITER", "ASSIGNED_TO_EDITOR");
+      break;
+    case "InProgress":
+      status.push("WRITING_IN_PROGRESS", "EDITING_IN_PROGRESS");
+      break;
+    default:
+      status;
+  }
+  const body = {
+    filters: { status }
+}
+const response = await fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=status`,
+       {
+        method: 'POST',
+        body: JSON.stringify(body),
+        headers: {
+          "Content-type": "application/json; charset=UTF-8"
+        }
+      }
+  );
+  const data = await response.json();
+  if(data?.success){
+    setStatus(data?.data?.uniqueValues)
+  }
+  if (!data?.success){
+    console.log("data>>",data?.error)
+	}
+}
+
+async function getAssignee(){
+const response = await fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=assignee`,
+       {
+        method: 'POST',
+        headers: {
+          "Content-type": "application/json; charset=UTF-8"
+        }
+      }
+  );
+  const data = await response.json();
+  if(data?.success){
+    setAssignee(data?.data?.uniqueValues.filter(Boolean))
+  }
+  if (!data?.success){
+    console.log("data>>",data?.error)
```

Comment:
Invoke the Toast component and pass this text.
"Something went wrong. Please try again"

Reply:
addressd